backend/tests/models/stub-db.js

```diff
@@ -0,0 +1,19 @@
+class DB {
+    insert(table, data) {
```

Comment:
Isn't the idea of the dependency injection that we can restore the database to a known state before each test?
What's idea behind this stub?